getting_started/5_Quantum_Teleportation.ipynb

```diff
@@ -0,0 +1,285 @@
+    "\n",
+    "\n",
+    "One important point to note is that at the end of the teleportation protocol, only Bob possses the quantum state $|\\psi\\rangle$, Alice no longer possesses the initial quantum state. \n",
+    "\n",
+    "\n",
+    "\n",
+    "![image.png](attachment:image.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Implementation in Amazon Braket\n",
+    "\n",
+    "Let's see how we can implement quantum teleportation protocol in Amazon Braket\n",
+    "\n",
+    "- Alice had the initial state to teleport in qubit-0. \n",
+    "- We'll create an entangled Bell Pair with qubit-1 and qubit-2. Alice has qubit-1 and Bob has qubit-2\n",
+    "- Alice applies CNOT between qubit-0 and qubit-1 and then a Hadamard to qubit-0\n",
+    "\n",
+    "- Depedning on what Alice's measures on qubit-0 & qubit-1, Bob has to apply these gates...\n",
+    "\n",
+    "|Alice's measurement|   Gates to apply|\n",
+    "|---|-----|\n",
+    "|00 | Nothing|\n",
+    "|01 |\tX|\n",
+    "|10 |\tZ|\n",
+    "|11 |\tZX|\n",
+    "\n",
+    "- To simulate measuring Alice's qubits and passing the measurement outputs to Bob classically - we'll use CNOT and CZ gates to Bob's qubit-2 - which achieves the same result as above table without performing partial measurements.\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from braket.circuits import Circuit\n",
+    "from braket.devices import LocalSimulator\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import time\n",
+    "%matplotlib inline\n",
+    "\n",
+    "device = LocalSimulator()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "T  : |0|1|2|3|4|5|6|\n",
+      "                    \n",
+      "q0 : -H-T-Z-H-C-H-C-\n",
+      "              |   | \n",
+      "q1 : -H-C-----X-C-|-\n",
+      "        |       | | \n",
+      "q2 : ---X-------X-Z-\n",
+      "\n",
+      "T  : |0|1|2|3|4|5|6|\n"
+     ]
+    }
+   ],
+   "source": [
+    "qt = Circuit()\n",
+    "\n",
+    "# Prepare Initial State ( you can experiment with different states)\n",
+    "\n",
+    "#qt.i(0)            # Initial State is 0\n",
+    "#qt.x(0)             # Initial State is 1\n",
+    "#qt.h(0)             # Initial State is a 50:50 superposition\n",
+    "#qt.h(0).t(0).h(0)     # Initial State is 85:15 superposition\n",
+    "qt.h(0).t(0).z(0).h(0) # Initial State is 15:85 superposition\n",
+    "#qt.ry(0,2*np.pi/5)      # Initial State is 66:33 superposition\n",
```

Comment:
We generally leave spaces after commas separating arguments in our style, e.g. "qt.ry(0, 2*np.pi/5)". Ditto below.

Reply:
updated all occurances